lib/grpc_web/client/client.rb

```diff
@@ -26,9 +26,9 @@ def initialize(base_url, service_interface)
 
   def define_rpc_method(rpc_method, rpc_desc)
     ruby_method = ::GRPC::GenericService.underscore(rpc_method.to_s).to_sym
-    define_singleton_method(ruby_method) do |params = {}|
+    define_singleton_method(ruby_method) do |params = {}, opt_headers = {}|
```

Comment:
Naming: can we call this `metadata` instead of `opt_headers` everywhere for consistency with grpc-ruby?

Example: https://github.com/grpc/grpc/blob/fed09cbe2c753af1c6717d06b0b3652beb46e055/src/ruby/lib/grpc/generic/service.rb#L169